_layouts/contribute.html

```diff
@@ -42,7 +42,8 @@ <h3>External Resources</h3>
               <li><a href="https://source.bazel.build/">Search Bazel code</a></li>
               <li><a href="/browse-and-search-user-guide.html">Search user guide</a></li>
               <li><a href="https://groups.google.com/forum/#!forum/bazel-dev">Developer mailing list</a></li>
-              <li><a href="irc://irc.freenode.net/bazel">IRC</a><li>
+              <li><a href="irc://irc.freenode.net/bazel">IRC</a></li>
```

Comment:
Shall we remove IRC links?

Reply:
I don't think we've agreed on deprecating IRC yet. I don't personally use it, but I believe some people do.

Reply:
Could we at least advocate one as the preferred venue to give clear guidance to otherwise dumbfounded visitors?

Reply:
Adding in @aiuto: since we've announced Slack at Bazelcon, does that mean that we're preferring it over IRC? AFAICT there is [only one thread on bazel-discuss on the Slack announcement](https://groups.google.com/forum/#!searchin/bazel-discuss/slack%7Csort:date/bazel-discuss/QJt9MRhrB-k/z29O1zO2AgAJ), and it didn't conclude that it's the official slack channel, and that (if) we are deprecating the IRC channel along with it.

Reply:
Let's just avoid a "so many Google chats - which one do I use? I give up" situation.

Reply:
We can add a note that the Slack channel is preferred and that some Bazel maintainers are on it, and the IRC channel isn't monitored. What do you think?

Reply:
SGTM.

We grabbed the slack channel exactly to avoid  "so many Google chat apps - which one do I use?".  This was to be the slack space where you were most likely to find a Bazel team member. But that was not a promise of SLO, nor an exclusion of anything else.
We never said we would not still monitor the bazel channels in other project's spaces. I know people lurk on those.

I've never looked at the IRC channel. I don't see an advantage to that over slack. The worst thing we can do is have multiple channels where the dev team communicates among itself. If people complain about no privacy on slack, that rings hollow if we want to do to development in the open.
One can always get person-to-person privacy by using mail.
If we want to keep a link to IRC, I would say "you might also find some team member on ..."

Reply:
done.